opm/simulators/wells/WellConvergence.hpp

```diff
@@ -30,13 +30,13 @@ namespace Opm
 
 class ConvergenceReport;
 class DeferredLogger;
-class WellInterfaceGeneric;
+template<class Scalar> class WellInterfaceGeneric;
 template<class Scalar> class WellState;
 
 class WellConvergence
 {
 public:
-    WellConvergence(const WellInterfaceGeneric& well)
+    WellConvergence(const WellInterfaceGeneric<double>& well)
```

Comment:
Same here: will this be changed in a later PR?

Reply:
will be changed later when WellConvergence gets its template param.